api/sys_rekey.go

```diff
@@ -55,8 +55,9 @@ func (c *Sys) RekeyUpdate(shard string) (*RekeyUpdateResponse, error) {
 }
 
 type RekeyInitRequest struct {
-	SecretShares    int `json:"secret_shares"`
-	SecretThreshold int `json:"secret_threshold"`
+	SecretShares    int      `json:"secret_shares"`
+	SecretThreshold int      `json:"secret_threshold"`
+	SecretPGPKeys   []string `json:"secret_pgp_keys"`
```

Comment:
I feel like `pgp_keys` would be better. The `secret_` of the others is referring to the secret sharing algorithm specifically